src/app_service/service/contacts/service.nim

```diff
@@ -301,6 +301,7 @@ QtObject:
         let msg = response.error.message
         error "error adding contact ", msg
         return
+      # FIXME: here i want to send message
```

Comment:
@saledjenic Here I should send a message, but in the current approach, there is no difference between just adding and requesting contact. Should I separate these implementations? 

Reply:
@MishkaRogachev my understanding is... there are 2 options for sending a contact request to anybody (but that's maybe more a question for @John-44):
1. from contact's profile popup, clicking on "Add to contact" button, in this case you're just sending a contact request, without identity verification request. If you want to send identity verification request later you should go to `profile -> messaging -> contacts` find that contact in appropriate section there and send identity verification request.
2. clicking on "Send contact request to chat key" button from `profile -> messaging -> contacts` section, which opens a search contact popup with a field to enter a message and that message will be the first step in identity verification process for contact you're adding. So in this second case sending a contact request combines two things, adding contact + running identity verification process.

Reply:
@MishkaRogachev for running identity verification process you should firstly check what is the state on the status-go side of that feature, cause you will need that for this ticket.

Reply:
Thanks, so probably should add sending message after identity verification process?

Reply:
@MishkaRogachev as we discussed on the last UI meeting on Friday, that message you're able to write when you're adding somebody to your contacts doesn't have anything with the "Identity Verification" process, don't look at it that way. That message is just here to "say hello..." to a contact you're adding. But now I am thinking maybe we should add that message as the first message in `1:1` chat with the contact you're adding once he accepts your contact request. Will ask that now in a `desktop-ui` channel.

Reply:
@saledjenic thank you for helping. I've asked about this mobile team and they have already implemented this in the status go with mutual contacts feature: https://github.com/status-im/status-go/pull/2665/files

Reply:
@MishkaRogachev then I am not sure if that is what we want to achieve. It's more a question for @John-44 now. :)